tests/manage/pv_services/pvc_snapshot/test_small_files_pvc_cloned_status.py

```diff
@@ -0,0 +1,140 @@
+    def test_pvc_status_after_clone(
+        self, pvc_factory, teardown_factory, snapshot_factory
+    ):
+        """
+        1. Create 1GiB PVC
+        2. Attach PVC to an application pod
+        3. Download Linux kernel and untar it to the directory where PVC is mounted
+        4. Create a new PVC from the snapshot
+        5. Take a snapshot of the PVC.
+        6. Create a new PVC out of that snapshot.
+        7. Verify PVC state
+
+        Args:
+            pvc_factory: A fixture to create new pvc
+            teardown_factory: A fixture to destroy objects
+            snapshot_factory: A fixture to create new snapshot
+        """
+        pvc_size = 1
+        worker_nodes_list = get_worker_nodes()
+        kernel_url = "https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.4.tar.xz"
+        download_path = "tmp"
+        dir_path = os.path.join(os.getcwd(), download_path)
+        file_path = os.path.join(dir_path, "linux-4.4.tar.xz")
+        if not os.path.exists(dir_path):
+            os.makedirs(dir_path)
+        urllib.request.urlretrieve(kernel_url, file_path)
+
+        # Create a RWX Cephfs PVC
+        pvc_obj = pvc_factory(
+            interface=constants.CEPHFILESYSTEM,
+            access_mode=constants.ACCESS_MODE_RWX,
+            size=pvc_size,
+        )
+
+        pvc_obj.reload()
+
+        # Create a pod on a particular node
+        selected_node = random.choice(worker_nodes_list)
+        log.info(f"Creating a pod on node: {selected_node} with pvc {pvc_obj.name}")
+
+        pod_obj = helpers.create_pod(
+            interface_type=constants.CEPHFILESYSTEM,
+            pvc_name=pvc_obj.name,
+            namespace=pvc_obj.namespace,
+            node_name=selected_node,
+            pod_dict_path=constants.PERF_POD_YAML,
+        )
+        teardown_factory(pod_obj)
+        log.info("Pod Teardown complete")
+
+        # Confirm that the pod is running on the selected_node
+        helpers.wait_for_resource_state(
+            resource=pod_obj, state=constants.STATUS_RUNNING, timeout=180
+        )
+        pod_obj.reload()
+
+        pod_name = pod_obj.name
+        pod_path = "/mnt"
+        _ocp = OCP(namespace=pvc_obj.namespace)
+
+        rsh_cmd = f"cp {dir_path} {pod_name}:{pod_path}"
+        _ocp.exec_oc_cmd(rsh_cmd)
+
+        rsh_cmd = f"exec {pod_name} -- tar xvf {pod_path}/tmp/linux-4.4.tar.xz -C {pod_path}/tmp"
+        _ocp.exec_oc_cmd(rsh_cmd)
+
+        # Create snapshots
+        log.info(f"Creating snapshot of PVC {pvc_obj.name}")
+        snap_obj = snapshot_factory(
+            pvc_obj=pvc_obj,
+            wait=True,
+            snapshot_name=f"{pvc_obj.name}-snapshot",
+        )
+        log.info("Snapshot created")
+
+        # Verify snapshots are Ready
+        log.info("Verify snapshot are ready")
+        snap_obj.ocp.wait_for_resource(
+            resource_name=snap_obj.name,
+            column=constants.STATUS_READYTOUSE,
+            timeout=180,
+        )
```

Comment:
You are calling the factory passing wait=True,  the factory will wait till the snapshot is ready.
These lines can be removed.